libc/test/src/__support/FPUtil/fpbits_test.cpp

```diff
@@ -8,215 +8,188 @@
 
 #include "src/__support/FPUtil/FPBits.h"
 #include "src/__support/FPUtil/fpbits_str.h"
+#include "src/__support/integer_literals.h"
```

Comment:
Are these some CMake changes that need to be made in addition to express this new dependency?